tests/test_browser.py

```diff
@@ -719,7 +719,8 @@ def test_sdl_image(self):
         ])
         self.run_browser('page.html', '', '/report_result?600')
 
-  def test_sdl_image_jpeg(self):
+  @also_with_wasmfs
+  def test_sdl_image_jpeg(self, wasmfs=False):
```

Comment:
```suggestion
  def test_sdl_image_jpeg(self):
```
Or is that needed for something?

Reply:
I need it for the `@also_with_wasmfs` decorator

Reply:
That seems surprising to me, since other decorators don't require it? We just add those decorator on a function and things work without changing the function.

Reply:
I did that because one of the tests that rely on the decorator is not fully run with `wasmfs` yet. So I had to inject a flag to cause that test to exit early when running with `wasmfs`.